shared/base/view.js

```diff
@@ -258,6 +258,15 @@ module.exports = BaseView = Backbone.View.extend({
       };
     }
 
+    // Allow ability to just pass the full "spec" to a lazy loaded view
+    if (this.options.fetch_spec) {
```

Comment:
Cool, thanks!  Well, this PR looks like it could be really useful to me.  :smile: